tsconfig.json

```diff
@@ -73,7 +73,8 @@
         "skipLibCheck": true, // Skip type checking of declaration files. ,
         "forceConsistentCasingInFileNames": true, // Disallow inconsistently-cased references to the same file.
 
-        "resolveJsonModule": true
+        "resolveJsonModule": true,
+        "noEmit": true
```

Comment:
I see you use `swc` as a compiler, so `noEmit` should be safe to use (avoids having TS generating js files)